tests/jobserver/api/test_releases.py

```diff
@@ -187,3 +188,74 @@ def test_upload_api_redirected(api_rf):
     release.refresh_from_db()
     assert response["Release-Id"] == release.id
     assert response["Location"] == f"http://testserver{release.get_absolute_url()}"
+
+
+@pytest.mark.django_db
+def test_release_index_api(api_rf):
```

Comment:
Not need for `api_rf` if you're using client here, might as well make it a fixture if we're using it in a few places too.

Does DRF's Client give us anything over a RequestFactory though?

Reply:
Yeah, I prefer Client() and APIClient() over request factory by default, as it checks that url is hooked up to the view. 

If you test everything with RF and calling the view directly, its possible to ship bad url config and break production. I have the scars :)

Reply:
Fair point!  Do you want to make a fixture for it to make `api_rf`?

Reply:
I didn't seem much point in a fixture, as there was nothing to clean up